.travis.yml

```diff
@@ -1,15 +1,15 @@
 language: android
+jdk: oraclejdk8
```

Comment:
By default, Travis uses Java 7, which causes the test suite to fail with "java.lang.UnsupportedClassVersionError: com/android/build/gradle/AppPlugin : Unsupported major.minor version 52.0".

Apparently Android Studio / gradle / Android 25 now requires Java 8. I confess I have ~no idea what requires what exactly here, and it might be a good idea to dig a bit deeper here as this probably goes beyond Travis (I get the same error on my local machine when using Java 7 instead of Java 8).